try{
    throw "Error de Programa"

    let x = noExiste()
}catch (error){
    alert(error)
}

let x = noExiste()